radicle-httpd/src/api/v1.rs

```diff
@@ -18,3 +18,21 @@ pub fn router(ctx: Context) -> Router {
 
     Router::new().nest("/v1", routes)
 }
+
+#[cfg(test)]
+mod utils {
+    use hyper::{body, Client, Uri};
+    pub use serde_json::json;
+
+    const BASE_URL: &str = "http://0.0.0.0:8080/api";
+    pub const HEAD: &str = "609769c057e54e63ef6e3220bb65118fb9bf0d5a";
+
+    pub async fn get_response(url: &str) -> serde_json::Value {
+        let uri = format!("{}{}", BASE_URL, url).parse::<Uri>().unwrap();
+        let response = Client::new().get(uri).await.unwrap();
+        let body_bytes = body::to_bytes(response.into_body()).await.unwrap();
+        let body = String::from_utf8(body_bytes.to_vec()).unwrap();
+
+        serde_json::from_str(&body).unwrap()
```

Comment:
We should be returning the status code as well (and checking it), and the headers just in case. Maybe create a `Response` type, or return a tuple.